Configs/.config/hypr/hyprland.conf

```diff
@@ -37,6 +37,7 @@ exec-once = dunst # start notification demon
 exec-once = wl-paste --type text --watch cliphist store # clipboard store text data
 exec-once = wl-paste --type image --watch cliphist store # clipboard store image data
 exec-once = ~/.config/hypr/scripts/swwwallpaper.sh # start wallpaper daemon
+exec-once = ~/.config/hypr/scripts/batterynotify.sh # battery notification
 #exec-once = swayidle -w timeout 900 'hyprctl dispatch dpms off' resume 'hyprctl dispatch dpms on' # turn off monitor after 15 mins
```

Comment:
can this be part of waybar module?

Reply:
No this is only for auto execution of script for dusnt notification for battery

Reply:
I meant to say why not make waybar execute this script every 5 mins, instead of making this a background process at launch?
For example, this change will cause a desktop to run this process 24x7 which is unnecessary...

Reply:
oh sorry i forget on next update i make waybar execute

Reply:
no worries, thank you so much!